src/libsync/bulkpropagatorjob.cpp

```diff
@@ -541,6 +541,7 @@ void BulkPropagatorJob::abortWithError(SyncFileItemPtr item,
 {
     abort(AbortType::Synchronous);
     done(item, status, error);
+    emit finished(status);
```

Comment:
@mgallien Why not put it inside `BulkPropagatorJob::done`? I can see other places where there is an error and `done` is called without subsequent emit to `finished`. Or, maybe other cases do not need `emit finished`?

Reply:
will check
not sure at the moment